schema.yaml

```diff
@@ -1088,6 +1088,55 @@ types:
         type: string
     type: object
     required: []
+  cloud-toolkit-aws:serverless:StaticWebArgs:
+    properties:
+      domain:
+        description: ""
```

Comment:
There are missing descriptions. I suggest finding which variables or classes miss a docstring and include one with a brief description.

Reply:
done